routes/api.php

```diff
@@ -23,9 +24,11 @@
 
     Route::name('bookings.reviews.assignable')->middleware(['permission:bookings.approve'])
         ->get('/bookings/assignable', [BookingReviewController::class, 'assignable']);
-
 });
 
+
+Route::post('/filterBookingRequests', [BookingRequestController::class, 'filter']);
```

Comment:
This should be behind the `auth:sanctum' guard. @a-drew has a fix incoming for this

Reply:
Yes I figured there was some sort of authentication needed, pushed this so we could discuss this in the PR. @a-drew will this fix be up by tonight/is it already up?

Reply:
the fix is already in main since my #507 PR